mesa/space.py

```diff
@@ -415,7 +415,8 @@ def place_agent(self, agent: Agent, pos: Coordinate) -> None:
 
     def remove_agent(self, agent: Agent) -> None:
         """Remove the agent from the grid and set its pos attribute to None."""
-        pos = agent.pos
+        if (pos := agent.pos) is None:
```

Comment:
This is definitely the first time the walrus operator is going to be used in the repo.

Reply:
I love it!! :)